yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -96,6 +96,11 @@ private[yarn] class YarnAllocator(
   // was lost.
   private val pendingLossReasonRequests = new HashMap[String, mutable.Buffer[RpcCallContext]]
 
+  // Executor loss reason for explicitly released executors, it will be added when executor loss
```

Comment:
Yes, the true. But at the initial test in my local environment, explicitly release executors will easily hit this issue. So I just commented like this, I will change it.